content/frameworks/react/getting-started/index.md

```diff
@@ -0,0 +1,97 @@
+
+<div class="aside caution">
+  The tutorials in this learning path do not focus on:
+
+  <ul>
+    <li>How to use React</li>
+    <li>How React works under the hood</li>
+  </ul>
+
+  Although both of these concepts will be touched on when needed, all the guides and codelabs in this section will instead focus on how to build fast and accessible React sites. For this reason, <strong>a basic knowledge of React is required.</strong>
+</div>
+
+To kick things off, this guide will briefly cover [Create React App](https://facebook.github.io/create-react-app/) which is used to build every application in this section. It will then conclude with a brief summary of all the different topics that will be explored in this learning path.
+
+## Create React App
+
+**Create React App** (CRA) is the easiest way to start building a React application with a build system that contains a module bundler (webpack) and transpiler (Babel). It takes care of these configurations by providing a default setup with a number of these core features baked in.
+
+On a command-line shell, you only need to run the following to create a new application+:
+
+```bash
+npx create-react-app new-app
+```
+
+<div class="aside note">
+<p><code>npx</code> is a package runner that is installed automatically with <code>npm</code> 5.2.0 or later. It simplifies quite a few processes involved with managing packages including running CLI commands (like <code>create-react-app</code> without having to install it globally on your machine.</p>
+
+<p>Take a look at <a href="https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b">Introducing npx: an npm package runner</a> if you would like to learn more.</p>
+</div>
+
+Once the command finishes executing, you can navigate to and run the application with the following commands:
+
+```bash
+cd new-app
+npm start
+```
+
+The following embed shows the directory structure and actual web page of a newly bootstrapped CRA application.
+
+<div class="glitch-embed-wrap" style="height: 346px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/new-create-react-app?path=src/App.js&attributionHidden=true"
+    alt="new-create-react-app on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+<div class="aside note">
+  Although CRA makes it simple to get started building a React application, there are many different ways to begin building sites with React. The <a href="https://github.com/facebook/create-react-app">"Popular Alternatives"</a> section within the README covers a few different cases where you might want to go with a different option.
```

Comment:
This is good. I like the way you've offered up alternatives.

Reply:
You could lean it a little bit-- how about this for the second sentence: "The "Popular Alternatives" section in the README covers different use-cases and options."

Reply:
Updated